syntax = "proto3";

package messages;

message BenchmarkMessage {
  string query = 1;
  int32 page_number = 2;
  int32 result_per_page = 3;
}
